ehsm_kms_service/delete_secret_thread.js

```diff
@@ -0,0 +1,137 @@
+const {
+    base64_decode
+} = require('./function')
+
+// Example of setInterval
+var init_interval
+
+// Record timer waiting to run
+var exist_timer = {}
+
+/**
+ * remove a timer from exist timer by key
+ * @param {Object} DB : database control
+ * @param {Object} key : key form exist timer
+ * @returns
+ */
+const removeTimer = (key, DB) => {
```

Comment:
Do you think both removeTimer and exist_timer requires to bind **1. secret name** together with **2. appid**?

Reply:
I think just need the secretName as the key enough